pkg/syncer/syncer.go

```diff
@@ -67,13 +75,34 @@ func StartSyncer(upstream, downstream *rest.Config, resources sets.String, clust
 		specSyncer.Stop()
 		return nil, err
 	}
+	leaseClient, err := clientset.NewForConfig(upstream)
+	if err != nil {
+		specSyncer.Stop()
+		return nil, err
+	}
+	leaseController := lease.NewController(
+		clock.RealClock{},
+		leaseClient,
+		fmt.Sprintf("cluster-%s", cluster),
+		int32(leaseDurationSeconds),
+		// TODO: onRepeatedHeartbeatFailure ? closeAllConns ?
```

Comment:
At least log a warning for now, and we can add more when we figure out what else we should do.

Reply:
✅